go.mod

```diff
@@ -29,16 +29,13 @@ require (
 	github.com/qeesung/image2ascii v1.0.1
 	github.com/rs/zerolog v1.33.0
 	github.com/saferwall/pe v1.5.3
-	github.com/santhosh-tekuri/jsonschema/v6 v6.0.1
-	github.com/santhosh-tekuri/jsonschema/v6 v6.0.1
+	github.com/santhosh-tekuri/jsonschema/v5 v5.3.1
```

Comment:
how come it downgraded here?

Reply:
i think its because v6 requires go 1.21 https://github.com/santhosh-tekuri/jsonschema/blob/v6.0.1/go.mod#L3. And i used go 1.19 to do a go mod tidy

Reply:
what should be the correct golang version to use? i was using 1.19 only because of https://github.com/kairos-io/kairos-sdk/blob/main/go.mod#L3 @mauromorales

Reply:
Go 1.19 is good in this case since it's the one on the go.mod, thanks @nianyush